tests/unit/test_validate_input.py

```diff
@@ -0,0 +1,29 @@
+import os
+import sys
+import tempfile
+import unittest
+
+from idseq_dag.steps.run_validate_input import PipelineStepRunValidateInput
+from idseq_dag.exceptions import InsufficientReadsError, InvalidFileFormatError
+
+fasta_filename = os.path.join(os.path.dirname(__file__), "fixtures", "reads.fasta")
+fastq_filename = os.path.join(os.path.dirname(__file__), "fixtures", "reads.fastq")
+
+class TestValidateInput(unittest.TestCase):
+    def test_validate_input(self):
+        PipelineStepRunValidateInput.quick_check_file(None, fasta_filename, is_fastq=False, max_fragments_to_check=100)
```

Comment:
What are these testing? No assertions?

Reply:
Testing that this works without error.

Reply:
Added an assertion on return value. Thanks for pointing it out.

Reply:
OK, Feels very weird to have two tests that would pass with an empty implementation of the function.

Reply:
I see, last response was before the last change.